projects/plugins/protect/src/js/components/interstitial-page/styles.module.scss

```
@@ -0,0 +1,13 @@
+.protect-header {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+    flex-wrap: wrap;
+}
+
+.get-started-button {
+
+	&:focus:not(:disabled) {
+		box-shadow: none;
+	}
+}
```

```suggestion
.get-started-button {
	&:focus:not(:disabled) {
		box-shadow: none;
	}
}

```
Very not important formatting suggestion 😅